vms/vms/settings.py

```diff
@@ -63,8 +63,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'vms',
-        'USER': 'vmsadmin',
-        'PASSWORD': '0xdeadbeef',
+        'USER': '',
+        'PASSWORD': '',
```

Comment:
@tapasweni-pathak Travis kept shooting an error due to these so I changed it. If it is required, I will make changes to create vmsadmin user in the yml file.